src/components/preferences/messages.js

```diff
@@ -3,6 +3,9 @@ export default {
     messages: {
         analysesViewShortcutLbl: "Open analyses view",
         analysisStatusEmailLbl: "Email me when my analysis status changes",
+        periodicEmailLbl: "Email me periodically when analyses are running",
+        periodicPeriodLbl:
+            "How frequently to notify about currently running jobs",
```

Comment:
This is probably an obsolete file (I think all our messages should be in `public/static/locales/en/`), but we can verify that later.